### DISCUSSION TITLE
password generator and netmask calculator

### DISCUSSION THREAD

**Discussion by htc1977** [CorentinTh/it-tools#285] [Ideas]
Hey, very nice toolset. There are only two things missing and it would be perfect for my usecases:

- a strong password generator like https://10015.io/tools/strong-random-password-generator
- a subnet calculator like https://www.solarwinds.com/free-tools/advanced-subnet-calculator

so far ... very good work, you've done!

**Comment by CorentinTh** [comment]
Is the [token generator](https://it-tools.tech/token-generator) not suitable for password generation ?

**Reply by CorentinTh** [reply]
Ah yeah maybe with the removal of confusing characters

**Reply by htc1977** [reply]
> Is the [token generator](https://it-tools.tech/token-generator) not suitable for password generation ?

Sorry, I didn't look at the string length ... it is perfect for my needs. :)
